EtherealEngine/etherealengine
# updateNeeded removed from channelSchema.ts

## Summary
- [x] `updateNeeded` removed from `channelSchema.ts`

## References
closes #9216 

## QA Steps
✔️ `Npm run check-errors` passed without problems
✔️ `Npm run check` passed with no errors as well
✔️ `Npm run dev` presents no issues
